client/src/Components/CreateNewRoom/CreateNewRoom.jsx

```diff
@@ -1,31 +1,33 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import axios from 'axios';
 
 import VoteButton from 'Components/VoteButton/index.js';
 import CreateNewRoomStyles from './CreateNewRoomStyles';
-import { useRoomContext } from 'Contexts/RoomContext';
+import ErrorProp from 'Components/ErrorBox/ErrorProp';
 
 const CreateNewRoom = () => {
   const history = useHistory();
   const classes = CreateNewRoomStyles();
-  const { setErrorMsg } = useRoomContext();
+  const [error, setError] = useState(undefined);
 
   const getRoomNumber = async () => {
     try {
       const currentRoom = await axios.post('/api/session');
-      if (currentRoom.data?.room?.id) history.push(`/room/${currentRoom.data.room.id}/join`, { isAdmin: true });
-      else {
-        setErrorMsg('Error : ' + currentRoom.data);
-      }
+      history.push(`/room/${currentRoom.data.room.id}/join`, { isAdmin: true });
     } catch (e) {
       console.error(e);
+      setError(e);
+      setTimeout(() => {
+        setError(undefined);
+      }, 6000);
     }
   };
   return (
     <>
       <div className={classes.buttonSession}>
         <VoteButton content={'START SESSION'} btnFunction={getRoomNumber}></VoteButton>
+        {error ? <ErrorProp content="Error: Not enough rooms" /> : null}
```

Comment:
Można wykorzystać sztuczkę (i często się to robi)
```suggestion
        {error && <ErrorProp content="Error: Not enough rooms" /> }
```